plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -154,7 +154,9 @@ class CodeModernizerSession(
         } catch (e: Exception) {
             state.putJobHistory(sessionContext, TransformationStatus.FAILED)
             state.currentJobStatus = TransformationStatus.FAILED
-            CodeModernizerStartJobResult.UnableToStartJob(e.message.toString())
+            CodeModernizerStartJobResult.UploadArtifactToS3Failure(
+                "https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/security_iam_manage-access-with-policies.html"
```

Comment:
Just pass in a normal String not the URL

Reply:
You can keep e.message.toString()

Reply:
We do not want to display the error message to user directly right ?